Sofa/Sofa/EventView.swift

```diff
@@ -0,0 +1,210 @@
+    private let awayScoreLabel = UILabel()
+    private let timeLabel = UILabel()
+    private let divider = UIImageView()
+    private let start = UILabel()
+    
+    private var event: Event
+    
+    init(event: Event) {
+        self.event = event
+        super.init()
+    }
+    
+    override func addViews() {
+        addSubview(homeTeamLogo)
+        addSubview(awayTeamLogo)
+        addSubview(homeTeamLabel)
+        addSubview(awayTeamLabel)
+        addSubview(homeScoreLabel)
+        addSubview(awayScoreLabel)
+        addSubview(timeLabel)
+        addSubview(divider)
+        addSubview(start)
+    }
+    
+    override func styleViews() {
+        backgroundColor = .white
+        homeTeamLogo.contentMode = .scaleAspectFit
+        awayTeamLogo.contentMode = .scaleAspectFit
+        
+        homeTeamLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        homeTeamLabel.textColor = .black
+        
+        awayTeamLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        awayTeamLabel.textColor = .black
+        
+        homeScoreLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        
+        awayScoreLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        
+        timeLabel.font = UIFont(name: "Roboto-Regular", size: 12)
+        timeLabel.textColor = .gray
+        timeLabel.textAlignment = .center
+        
+        start.font = UIFont(name: "Roboto-Regular", size: 12)
+        start.textColor = .gray
+        start.textAlignment = .center
+
+        
+        divider.contentMode = .scaleAspectFit
+        divider.image = UIImage(named: "Divider Horizontal")
+        
+        if let homeLogoUrl = event.homeTeam.logoUrl, let url = URL(string: homeLogoUrl) {
+            homeTeamLogo.loadImage(from: url)
+        }
+        if let awayLogoUrl = event.awayTeam.logoUrl, let url = URL(string: awayLogoUrl) {
+            awayTeamLogo.loadImage(from: url)
+        }
+        
+        homeTeamLabel.text = event.homeTeam.name
+        awayTeamLabel.text = event.awayTeam.name
+        
+        
+        if let homeScore = event.homeScore {
+            homeScoreLabel.text = "\(homeScore)"
+        }
+
+        if let awayScore = event.awayScore {
+            awayScoreLabel.text = "\(awayScore)"
+        }
+        
+        let (timeText, timeColor,Live) = calculateMatchTime(startTimestamp: event.startTimestamp)
+        timeLabel.text = timeText
+        timeLabel.textColor = timeColor
+        
+        if Live {
+            homeScoreLabel.textColor = .blue
+            awayScoreLabel.textColor = .blue
+        } else {
+            homeScoreLabel.textColor = .black
+            awayScoreLabel.textColor = .black
+        }
+        
+        let formatter = DateFormatter()
+        formatter.dateFormat = "HH:mm"
+
+        let startDate = Date(timeIntervalSince1970: TimeInterval(event.startTimestamp))
+        start.text = formatter.string(from: startDate)
```

Comment:
Pošto ne želimo na Viewu imati nikakvu logiku, želimo da je view maksimalno glup, odnonso želimo dobiti čiste vrijednosti/kalkulirane vrijednosti koje prezentiramo na view.

Znači da možeš ovdje pozoveš samo
startLabel.text = event.startDateText (ili model.startDateText)